packages/widget_toolkit_pin/lib/src/lib_pin_code_with_biometrics/blocs/pin_code_bloc.dart

```diff
@@ -20,17 +20,18 @@ abstract class PinCodeBlocEvents {
 
   ///Triggered when user tap on biometrics button
   void biometricsButtonPressed();
-
-  void checkIfPinIsStored();
 }
 
 /// A contract class containing all states of the PinCodeBloc.
 abstract class PinCodeBlocStates {
-  ///Indicating how much dots to present
+  /// Emits cyrrent PIN length to UI
```

Comment:
Typo in word `cyrrent` (should be `ccurrent`)

Reply:
Good catch! Dart Analysis couldn't save me from that 😆